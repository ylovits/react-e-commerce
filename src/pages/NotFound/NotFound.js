import React from "react";
import { Link } from "react-router-dom";

const NotFound = () => {
  return (
    <div className="container d-flex flex-column h-100 flex-grow-1">
      <div className="row flex-grow-1 align-items-center text-center">
        <div className="col">
          <h1>Page Not Found</h1>
          <span>
            Return to the <Link to="/">Home page</Link>
          </span>
        </div>
      </div>
    </div>
  );
};

export default NotFound;
