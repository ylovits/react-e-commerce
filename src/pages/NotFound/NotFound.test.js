import React from "react";
import { shallow } from "enzyme";
import NotFound from "./NotFound";

it("<NotFound /> rendered", () => {
  expect(shallow(<NotFound />)).toMatchSnapshot();
});
