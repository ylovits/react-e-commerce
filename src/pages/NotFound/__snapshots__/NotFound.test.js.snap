// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> rendered 1`] = `ShallowWrapper {}`;
